### ISSUE TITLE
`koverMergedHtmlReport` task fails due `classInstructions` being null

### ISSUE THREAD

**Issue by JavierSegoviaCordoba** [Kotlin/kotlinx-kover#296]
**Describe the bug**
Running `./gradlew koverMergedHtmlReport` fails.

**Errors**
```
> Task :koverMergedHtmlReport FAILED
Exception in thread "main" java.lang.NullPointerException: Cannot invoke "com.intellij.rt.coverage.data.instructions.ClassInstructions.getlines()" because "classInstructions" is null
        at com.intellij.rt.coverage.util.InstructionsSection.saveClass(InstructionsSection.java:96)
        at com.intellij.rt.coverage.util.ClassListSection$1.execute(ClassListSection.java:51)
        at com.intellij.rt.coverage.util.ClassListSection$1.execute(ClassListSection.java:48)
        at org.jetbrains.coverage.gnu.trove.TObjectIntHashMap.forEachEntry(TObjectIntHashMap.java:379)
        at com.intellij.rt.coverage.util.ClassListSection.saveInternal(ClassListSection.java:48)
        at com.intellij.rt.coverage.util.ReportSection.save(ReportSection.java:61)
        at com.intellij.rt.coverage.util.ReportSectionsUtil.saveSections(ReportSectionsUtil.java:74)
        at com.intellij.rt.coverage.instrumentation.SaveHook.save(SaveHook.java:103)
        at com.intellij.rt.coverage.aggregate.Aggregator.processRequests(Aggregator.java:98)
        at com.intellij.rt.coverage.aggregate.Main.main(Main.java:27)
```

**Environment**
 - Kover Gradle Plugin version: 0.6.1
 - Gradle version: 7.6
 - Kotlin project type: Kotlin/JVM
 - Coverage Engine version (if customized in build script): Defaults
 - Other context important for this bug: All OS

**Comment by JavierSegoviaCordoba** [comment]
@shanshin where can I find the latest version or the version history for the IntelliJ coverage engine?

**Comment by zuevmaxim** [comment]
@JavierSegoviaCordoba  Please try a fix with `1.0.706` engine version:
for Kotlin Gradle script
```
kover {
    engine.set(kotlinx.kover.api.IntellijEngine("1.0.706"))
}
```
for Groovy
```
kover {
    engine = new kotlinx.kover.api.IntellijEngine("1.0.706")
}
```
**Could you please share files from `build/kover/errors` after you run your tests with kover?**

You can find IntelliJ Engine version history here: https://central.sonatype.com/artifact/org.jetbrains.intellij.deps/intellij-coverage-agent/1.0.706/overview
